app/src/hooks/useTeamManage.ts

```diff
@@ -0,0 +1,66 @@
+import { useMutation, useQuery, useQueryCache } from 'react-query';
+
+import { QUERY_TEAM_KEY, QUERY_TEAM_MEMBERS } from '../const/query.const';
+import { useAxios } from './useAxios';
+import { UserProfile } from './useUserProfile';
+
+export interface TeamProfile {
+  id: number;
+  name: string;
+  description: string;
+  owner_id: number;
+  updated_at: string;
+  created_at: string;
+}
+
+interface TeamExit {
+  team_id: number;
+  user_id: number;
+}
+
+export type TeamMembersResponse = UserProfile[];
+
+export const useTeamProfileQuery = () => {
+  const axios = useAxios();
+
+  return useQuery<TeamProfile>(QUERY_TEAM_KEY, async () => {
+    const response = await axios.get<TeamProfile>(QUERY_TEAM_KEY);
+    return response.data;
+  });
+};
+
+export const useShowMembersOfTeamQuery = (
+  team_id: number | null | undefined
+) => {
+  const axios = useAxios();
+
+  return useQuery<TeamMembersResponse>(
+    QUERY_TEAM_MEMBERS,
+    async () => {
+      if (team_id === undefined) {
+        return [];
+      }
```

Comment:
Ten if jest niepotrzebny, ze względu na to, że w opcjach Query określamy je jako zależne od parametru `team_id`, więc dopóki zwraca on falsy value to hook nie zostanie wykonany. Do usunięcia.